# MeshCat Visualizer
mesh_dir : "/home/julius/devel/RoboDemos/ROS2_packages/panda2_description/panda/meshes"
urdf_model_path : "/home/julius/devel/RoboDemos/ROS2_packages/panda2_description/urdf/panda2_inertias.urdf"
# Waypoints Node
waypoints_dir : "/home/julius/dumps/waypoints.csv"
traces_dir_tmpl : "/home/julius/dumps/traces_{number}.csv"